src/lib/menu/menu-trigger.ts

```diff
@@ -283,7 +305,14 @@ export class MdMenuTrigger implements AfterViewInit, OnDestroy {
   // set state rather than toggle to support triggers sharing a menu
   private _setIsMenuOpen(isOpen: boolean): void {
     this._menuOpen = isOpen;
-    this._menuOpen ? this.onMenuOpen.emit() : this.onMenuClose.emit();
+
+    if (this._menuOpen) {
+      this.onMenuOpen.emit();
```

Comment:
Rather than having to emit to multiple subjects, why not have something like:

```
@Output() onMenuOpen = new EventEmitter<void>();
@Output() onMenuOpened = this.onMenuOpen;
```

Reply:
Thanks for the feedback! Done :)